src/python/zquantum/core/bitstring_distribution/distance_measures/mmd.py

```diff
@@ -62,14 +62,13 @@ def compute_multi_rbf_kernel(x_i: np.array, y_j: np.array, sigmas: List) -> np.n
             np.ndarray: The gaussian kernel matrix.
     """
     exponent = np.abs(x_i[:, None] - y_j[None, :]) ** 2
-    kernel_matrix = 0.0
     for sigma in sigmas:
         try:
             gamma = 1.0 / (2 * sigma)
         except ZeroDivisionError as error:
             print("Handling run-time error:", error)
             raise
-        kernel_matrix = kernel_matrix + np.exp(-gamma * exponent)
+        kernel_matrix = np.exp(-gamma * exponent)
```

Comment:
I think you have changed the logic here. Shouldn't it be `+=`?

Reply:
Ah you're right. I had seen that `kernel_matrix` was being initialized to zero, but didn't see that the addition was happening in a loop.

Reply:
fixed in 2d5abfa

Reply:
Power of PRs!